public/model/searched-admins.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Which admins the user has left in search box.
+ */
+
+var P = require('pjs').P;
+
+var SearchedAdmins = P({
+  init: function(onUpdate) {
+    this.onUpdate = onUpdate;
+    // `searched_admin_codes` is a map from admin code to callbacks. The
+    // callbacks are called when the admin is unsearched.
+    this.searched_admin_codes = {};
+    // Temp hack to avoid double refocus on search select
```

Comment:
Yeah, try to use the event emitter stuff ... this is kind of a perfect use case.